test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/util.py

```diff
@@ -346,3 +357,51 @@ def create_and_invoke_endpoint(
             return invoke_endpoint(
                 sagemaker_runtime_client, model_name, input_data, target_models, content_type
             )
+
+
+def create_and_invoke_endpoint_helper(
```

Comment:
Can we host this function [here](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/__init__.py)?

Reply:
this is a wrapper around the existing `create_and_invoke_endpoint` function which was already in this location. Do we want all of those moved?

Reply:
if you look closely in the [invoke_sm_helper_function](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/__init__.py#L71) function parameters,[test_function](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/__init__.py#LL86C12-L86C25) accepts function as an input, hence, `create_and_invoke_endpoint` used here.

I would suggest to either use the  [invoke_sm_helper_function](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/__init__.py#L71) or duplicate it to suite your needs in the same init file and use it in the pytest function